MAINTAINERS.yml

```diff
@@ -321,6 +321,16 @@ CMSIS API layer:
     - "area: CMSIS API Layer"
     - "area: Portability"
 
+DSP subsystem:
+  status: maintained
+  maintainers:
+    - yperess
```

Comment:
Please add me as a maintainer as well. I would be interested in any changes to the CMSIS-DSP backend and the ZDSP tests that have been relocated from the CMSIS-DSP tests.

Reply:
Done